ILRepack/RepackOptions.cs

```diff
@@ -97,6 +97,8 @@ public List<string> AllowedDuplicateNameSpaces
             get { return allowedDuplicateNameSpaces; }
         }
 
+		public string RepackDropAttribute { get; set; }
```

Comment:
indent is off by 1 here

Reply:
Yep, you are right. That file needs reformatting.

Reply:
OK, moved `hasFilter` check into `ShouldDrop`